src/inc/til/point.h

```diff
@@ -64,7 +64,7 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             return !(*this == other);
         }
 
-        operator bool() const noexcept
+        explicit operator bool() const noexcept
         {
             return _x != 0 || _y != 0;
         }

```

Comment:
@miniksa Wanted to run an idea by you about this, actually. Maybe we should just remove the bool operator altogether from Point because 0, 0 is actually valid.

Reply:
That's fine, then chuck it.